### PR TITLE
Remove RxAtomic leftovers

### PR DESCRIPTION
**Pull request by freak4pc**
Hey @kzaher. It seems there were a couple of leftovers when removing RxAtomic from the project, wanted to clean this up.

### CODE REVIEW

**Comment by freak4pc** [comment]
Uhm, are we still using RxAtomic on Linux?

**Comment by kzaher** [comment]
> Uhm, are we still using RxAtomic on Linux?

We shouldn't.

File: Tests/RxSwiftTests/Atomic+Overrides.swift [hunk 1]
```diff
@@ -1,5 +1,5 @@
 //
```
Comment (kzaher) [inline Tests/RxSwiftTests/Atomic+Overrides.swift]: You can delete this entire file and all depending on it.

Comment (freak4pc) [reply Tests/RxSwiftTests/Atomic+Overrides.swift]: It's used in tests - should I just directly use the implementation ?

**Pull request merged**